plugins/ssr/ssrelim.ml

```diff
@@ -213,8 +202,6 @@ let ssrelim ?(is_case=false) deps what ?elim eqid elim_intro_tac =
             Array.mapi (fun j (ctx, cty) ->
               let t = Term.it_mkProd_or_LetIn cty ctx in
                     ppdebug(lazy Pp.(str "Search" ++ Printer.pr_constr_env env (project gl) t));
-              let t = Arguments_renaming.rename_type t
-                (GlobRef.ConstructRef((kn,i),j+1)) in
```

Comment:
Here ssr renames the nf_lc from the inductive. If I rename in nf_lc the names generated by destruct change which breaks the stdlib, see eg https://gitlab.com/coq/coq/-/jobs/381949496
I don't know how important this ssr renaming is.

Reply:
It is a new feature, the `=> [^ prefix]` one that uses these names.

I'm ok keeping a call to rename here.